src/panels/config/devices/device-detail/integration-elements/zwave_js/device-actions.ts

```diff
@@ -27,13 +32,19 @@ export const getZwaveDeviceActions = async (
 
   const entryId = configEntry.entry_id;
 
-  const node = await fetchZwaveNodeStatus(hass, device.id);
+  const results = await Promise.all([
```

Comment:
```suggestion
  const [nodeStatus, firmwareCapabilities] = await Promise.all([
```

Reply:
thanks I couldn't figure out how to do this, didn't realize I could put the `const` at the front